### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by zhouhao3**

### CODE REVIEW

File: cluster-up/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # How to use cluster-up
 
-This directory provides a wrapper around gocli. It can be vendored into other
+This directory provides a wrapper around gocli. It can be vended into other
```
Comment (brianmcarey) [inline cluster-up/README.md:3]: I think `vendored` is ok here - the rest look good to me.

Comment (zhouhao3) [reply cluster-up/README.md]: updated, thanks.

**Review by brianmcarey — APPROVED** [review]
/lgtm

/approve

**Comment by oshoval** [comment]
Thanks for the PR

@dhiller  @brianmcarey 
Note please that with provision manager we won't rebuild provider when md files are changed
moreover since less providers would be built, less jobs are prone to fail as they now fail and the whole 3h might have to be redone

**Pull request merged**